pkg/infra/iac2/templates_compiler.go

```diff
@@ -110,6 +124,24 @@ func (tc templatesCompiler) RenderImports(out io.Writer) error {
 	return nil
 }
 
+func (tc templatesCompiler) RenderPackageJSON() (javascript.NodePackageJson, error) {
+	errs := multierr.Error{}
+	mainPJson := javascript.NodePackageJson{}
+	for _, res := range tc.resourceGraph.GetAllVertices() {
+		pJson, err := tc.GetPackageJSON(res)
+		if err != nil {
+			fmt.Println(err)
+			errs.Append(err)
+			continue
+		}
+		mainPJson.Merge(&pJson)
+	}
+	if err := errs.ErrOrNil(); err != nil {
+		return *mainPJson.Clone(), err
+	}
+	return *mainPJson.Clone(), nil
```

Comment:
No need for clone here since `mainPJson` isn't used anywhere else.

Reply:
yeah but because it contains a mux i believe go static check yells at you if you return it directly